portfolio/src/main/webapp/aboutme.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Akin's Portfolio</h1>
+      <div id="navigation">
+        <h3><a href='index.html'>HOME</a></h3>
```

Comment:
nit: I recommend using the same quotes throughout the whole file. Typically people will use double quotes, but as long as it's consistent there's no issue.